resources/deployments/controller/controller_deployments.go

```diff
@@ -173,8 +173,12 @@ func (d *DeploymentsController) AbortDeployment(w rest.ResponseWriter, r *rest.R
 	d.view.RenderEmptySuccessResponse(w)
 }
 
+const (
+	GetDeploymentForDeviceQueryArtifact   = "artifact_name"
+	GetDeploymentForDeviceQueryDeviceType = "device_type"
```

Comment:
Hmm, never mind. I thought it would be used to compare against the artifact file that the device type is correct, but I guess that's not implemented yet.